Makefile

```diff
@@ -28,7 +34,9 @@ format-style:
 	autopep8 `git ls-files | grep "\.py$$"`
 
 test: clean check-syntax-errors check-style check-types
-	env PYTEST_ADDOPTS="$$PYTEST_ADDOPTS --cov=openfisca_core" pytest
+	PYTEST_ADDOPTS="$$PYTEST_ADDOPTS ${pytest_addopts}" pytest
+	openfisca test ${COUNTRY_TEMPLATE_TESTS} -c ${COUNTRY_TEMPLATE} ${optional_arguments}
+	openfisca test ${EXTENSION_TEMPLATE_TESTS} -c ${COUNTRY_TEMPLATE} -e ${EXTENSION_TEMPLATE} ${optional_arguments}
 
-api:
-	openfisca serve --country-package openfisca_country_template --extensions openfisca_extension_template
+serve:
```

Comment:
Are we sure this should not be a breaking change if we change the `make` target name? What if some user automates with `make api`?

> I'm still in favour of this rename, but I believe we should either add an alias or make this a breaking change (or drop this target altogether because I don't understand what's the point of `make test` vs `openfisca test`).